# Generated by Django 4.1.3 on 2022-12-10 17:50

from django.conf import settings
import django.contrib.auth.validators
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        ('sweetusers', '0020_alter_subscriptions_object_and_more'),
    ]

    operations = [
        migrations.RemoveField(
            model_name='subscriptions',
            name='object',
        ),
        migrations.RemoveField(
            model_name='subscriptions',
            name='subject',
        ),
        migrations.AlterField(
            model_name='user',
            name='username',
            field=models.CharField(default='User1670694642', error_messages={'unique': 'A user with that username already exists.'}, help_text='Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.', max_length=150, unique=True, validators=[django.contrib.auth.validators.UnicodeUsernameValidator()], verbose_name='username'),
        ),
        migrations.AddField(
            model_name='subscriptions',
            name='object',
            field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, related_name='subscriptions', to=settings.AUTH_USER_MODEL),
        ),
        migrations.AddField(
            model_name='subscriptions',
            name='subject',
            field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, related_name='author_subscription', to=settings.AUTH_USER_MODEL),
        ),
    ]
